java-sdk/radar-schemas-tools/src/main/java/org/radarcns/schema/specification/DataProducer.java

```diff
@@ -26,9 +26,6 @@
     @JsonProperty
     private List<String> labels;
 
-    @JsonProperty
-    private String version;
```

Comment:
Why not add the other two properties here, instead of adding them to the subclasses?

Reply:
The reason i moved it to AppSource is having those in the `DataProducer` also affects `StreamGroups`. I am not sure whether we want that?. @blootsvoets